rasa/nlu/training_data/formats/markdown.py

```diff
@@ -55,13 +53,13 @@ class MarkdownReader(TrainingDataReader):
     """Reads markdown training data and creates a TrainingData object."""
 
     def __init__(self) -> None:
-        self.current_title = None
-        self.current_section = None
-        self.training_examples = []
-        self.entity_synonyms = {}
-        self.regex_features = []
+        self.current_title: Optional[Text] = None
```

Comment:
have we already dropped Python 3.5 supports? Type annotation for variables are not supported in Py 3.5

Reply:
good point, forgot about it, will remove them again.